src/sentry/utils/jwt.py

```diff
@@ -0,0 +1,64 @@
+"""Common handling of JWT tokens.
+
+This is an attempt to have all the interactions with JWT in once place, so that we have once
+central place which handles JWT in a uniform way.
+"""
+
+from typing import Mapping, Union
+
+import jwt
+from jwt import DecodeError
+
+__all__ = ["peek_claims", "decode", "encode", "authorization_header", "DecodeError"]
+
+
+def peek_claims(token: str) -> Mapping[str, str]:
+    """Returns the claims (payload) in the JWT token without validation.
+
+    These claims can be used to look up the correct key to use in :func:`decode`.
+    """
+    return jwt.decode(token, verify=False)
+
+
+def decode(token: str, key: bytes, verify_aud: bool = True) -> Mapping[str, str]:
```

Comment:
I'm not really sure about this choice of `verify_aud`.  Perhaps it is better to just accept `options` and pass it through to PyJWT.

Reply:
I like the approach of making options into kwargs like this. It constrains the user's options and picks sane defaults.